server/src/com/cloud/network/element/VpcVirtualRouterElement.java

```diff
@@ -482,8 +474,8 @@ public boolean applyIps(final Network network, final List<? extends PublicIpAddr
         if (canHandle) {
             final List<DomainRouterVO> routers = _routerDao.listByNetworkAndRole(network.getId(), Role.VIRTUAL_ROUTER);
             if (routers == null || routers.isEmpty()) {
-                s_logger.debug(getName() + " element doesn't need to associate ip addresses on the backend; VPC virtual " + "router doesn't exist in the network "
-                        + network.getId());
+                s_logger.debug(
+                        getName() + " element doesn't need to associate ip addresses on the backend; VPC virtual " + "router doesn't exist in the network " + network.getId());
```

Comment:
While this change is the result of a reformat, would it be possible to wrap this logging statement in a ``s_logger.isDebugEnabled()`` check to avoid unnecessary string concatenation when ``DEBUG`` logging is not enabled?